app/views/drivers/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if @driver.errors.any? %>
```

Comment:
I suggest showing the errors and the form **both** if there are errors, otherwise they have to go back to the edit page manually.